brion/plugin/morphologySWC.cpp

```diff
@@ -36,12 +36,25 @@ namespace
 {
 
 // It's not clear if -1 is the only way of identifying a root section.
-const int UNDEFINED_PARENT_SWC = -1;
+const int SWC_UNDEFINED_PARENT = -1;
+
+enum SWCSectionType
+{
+    SWC_SECTION_UNDEFINED = 0,
+    SWC_SECTION_SOMA,
+    SWC_SECTION_AXON,
+    SWC_SECTION_DENDRITE,
+    SWC_SECTION_APICAL_DENDRITE,
+    SWC_SECTION_FORK_POINT = 5,
```

Comment:
I wanted to make it easy to identify which number corresponds to each tag. Instead of writing down all the value I just wrote that value (and now I'm writing longer to give explanations).